templates/beta-release-post.adoc

```diff
@@ -53,19 +53,11 @@ BETA_RELEASE_SUMMARY
 // // // // // // // //
 // Change the RELEASE_SUMMARY to an introductory paragraph. This sentence is really
 // important becuase it is supposed to grab the readers attention.  Make sure to keep // the blank lines 
+//
+// Throughout the doc, replace RELEASE_VERSION with the version number of Open Liberty, eg: 22.0.0.2-beta
 // // // // // // // //
-We have two beta packages for link:{url-about}[Open Liberty]:
-
-* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
-* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
-
-This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
-
-If you try either package, <<feedback, let us know what you think>>.
-[#allbeta]
-== All Beta Features package
 
-The All Beta Features package includes the following beta features:
+The link:{url-about}[Open Liberty] RELEASE_VERSION includes the following beta features (along with all GA features):
```

Comment:
```suggestion
The link:{url-about}[Open Liberty] RELEASE_VERSION includes all GA features and the following beta features:
```

Reply:
I think I'm going to keep this as-is, as I'd like the focus to be on the beta features (and the GA portion is more of an 'fyi')